### PR TITLE
Graceful fails

### PR DESCRIPTION
**Pull request by seunomonije**
now fails gracefully in almost every case. Still looking for more
Also fixed bug with instants in cache.

### CODE REVIEW

File: client/css/analysis.css [hunk 1]
```diff
@@ -324,3 +324,46 @@ h4 > a:hover {
 #opening-wrapper > a {
   text-decoration: none;
 }
+
+/* Graceful-fails */
+.searchbarglow {
+  animation: shake 0.6s, invalidSearch 4s ease-in-out 0s alternate 1;
+}
```
Comment (gracetian6) [inline client/css/analysis.css:331]: bougie!!

File: client/css/analysis.css [hunk 2]
```diff
@@ -324,3 +324,46 @@ h4 > a:hover {
...
+    background-color: #d3d3d3 /*light-gray*/
+  }
+  
+  20% {
+    background-color: #db3236; /* Google red */
+  }
+
+  100% {
+    background-color: #d3d3d3 /*light-gray*/
+  }
```
Comment (gracetian6) [inline client/css/analysis.css:344]: this animation is so cool

File: client/js/analysis.js [hunk 3]
```diff
@@ -317,3 +322,13 @@ function showLightbox() {
...
 
+/**
+ * Shakes the search bar and make it glow
+ */
+function shakeSearchBar() {
+  const el = document.getElementById('searchbar');
+  el.classList.add('searchbarglow');
+  el.onanimationend = () => {
+    el.classList.remove('searchbarglow');
+  };
```
Comment (gracetian6) [inline client/js/analysis.js:333]: Yessss

File: client/js/analysis.js [hunk 4]
```diff
@@ -317,3 +322,13 @@ function showLightbox() {
   target.classList.remove('hide-lightbox');
 }
 
+/**
+ * Shakes the search bar and make it glow
+ */
```
Comment (gracetian6) [inline client/js/analysis.js:327]: small nit: add that this is for errors
`Shakes the search bar and make it glow for user error`

File: client/js/analysis.js [hunk 5]
```diff
@@ -1,6 +1,16 @@
 const COMMENT_APPEARANCE_TIME = 1500;
 const COMMENT_TO_STOP_AT = 5;
 const FEEDBACK_APPEARANCE_TIME = 1500;
+const COOLDOWN_TIME = 1000;
+const TEN_MINUTES_IN_SECONDS = 3600;
```
Comment (jessicaslaughter) [inline client/js/analysis.js:5]: there are 600 seconds in 10 minutes, not 3600. also you should replace the number with the equation that resulted in that number (60 * 10)

File: client/js/analysis.js [hunk 6]
```diff
@@ -25,6 +37,10 @@ async function fetchResponse() {
     console.error(e);
   }
   if (response) {
+    if (response.status == 500) {
+      alert('The video you selected is incompatible with this tool. Please try again.');
```
Comment (geekster777) [inline client/js/analysis.js:41]: When would you expect this to happen? Leave a comment.

**Pull request merged**